src/main/rules/GCI604/java/GCI604.asciidoc

```diff
@@ -0,0 +1,31 @@
+Spring Retry: Inconsistent configuration
+= Inconsistent Spring Retry Configuration
+
+An inconsistent configuration of Spring Retry can lead to blocking threads and increased energy usage.
```

Comment:
could you please add some references like : 
- https://medium.com/@adriancanton/spring-retry-implementing-retries-in-your-spring-boot-applications-43dd4c52f93f
- https://dzone.com/articles/implementing-exponential-backoff-with-spring-retry

if we respect our process for a new rule ... we have to give evidence to prove that its a good idea like : 
- web documentation
- performance tests (on web documentation or local benchmark)
- link on the rule idea inside green-code-initiative : I think this is this one, isn't it ? https://github.com/green-code-initiative/creedengo-challenge/issues/78